lib/gooddata/lcm/actions/base_action.rb

```diff
@@ -101,6 +101,23 @@ def add_failed_segment(segment_id, message, error_action, params)
           end
         end
 
+        def add_project_client_mapping(project_id, client_id, segment_id, params)
```

Comment:
Please add document to new function if needed

Reply:
Done add document